extensions/omni_httpd/tests/create_drop.yml

```diff
@@ -3,12 +3,21 @@ instance:
   config:
     shared_preload_libraries: */env/OMNI_EXT_SO
     max_worker_processes: 64
+  init:
+    - create extension omni_vfs cascade
 
 tests:
 
 - name: shuts down after drop
   tests:
   - query: create extension omni_httpd cascade
     commit: true
+
+    # https://github.com/omnigres/omnigres/issues/479
+  - query: select count(*)
+           from omni_vfs.list(omni_vfs.local_fs((select setting from pg_settings where name = 'data_directory')), '/')
```

Comment:
I think here in the text we can get away with `pg_ls_dir` instead of omni_vfs?

Reply:
Wasn't aware of it, have replaced omni_vfs with pg_ls_dir, thanks for pointing out